apps/site/src/app/(home)/sections/Countdown/Countdown.tsx

```diff
@@ -0,0 +1,218 @@
+"use client";
+
+import bg_map from "@/assets/images/maps/countdown.svg";
+import Image from "next/image";
+import useWindow from "./useWindow";
+
+import styles from "./Countdown.module.scss";
+import CountdownClock from "./CountdownClock";
+import { useEffect, useState } from "react";
+
+import boat from "@/assets/icons/boat.png";
+
+interface CountdownProps {
+	schedule: {
+		title: string;
+		location?: string | undefined;
+		virtual?: string | undefined;
+		startTime: Date;
+		endTime: Date;
+		organization?: string | undefined;
+		hosts?: string[] | undefined;
+		description: any;
+	}[];
+}
+
+const Countdown: React.FC<CountdownProps> = ({ schedule }) => {
+	const hackStartTime = new Date("2024-11-02T10:00:00"); // TBD, zothacks start time
+	const hackEndTime = new Date("2024-11-02T22:00:00"); // TBD, zothacks end time
+
+	const [curTime, setCurTime] = useState(new Date());
+
+	useEffect(() => {
+		const i = setInterval(() => {
+			setCurTime(new Date());
+		}, 1000);
+
+		return () => clearInterval(i);
+	}, []);
+
+	const ended = schedule.filter((el) => el.endTime >= curTime);
+
+	const before =
+		ended.length > 0
+			? ended[0]
+			: {
+					title: "That's a wrap~",
+					location: "",
+					startTime: new Date(0),
+					endTime: new Date(0),
+				};
+
+	const after =
+		ended.length > 1
+			? ended[1]
+			: {
+					title: "That's a wrap~",
+					location: "",
+					startTime: new Date(0),
+					endTime: new Date(0),
+				};
+
+	const percentageCrossed =
+		before.endTime.getTime() > 0
+			? curTime.getTime() < before.startTime.getTime()
+				? 0
+				: ((before.endTime.getTime() - curTime.getTime()) /
+						(before.endTime.getTime() - before.startTime.getTime())) *
+					100
+			: 100;
+
+	const [w, h] = useWindow();
+
+	const totalLines = Math.floor(w / 66) > 7 ? Math.floor(w / 66) : 7;
+
+	const totals = Array(totalLines + 1).fill(0);
+
+	function returnPosition(num: number) {
+		// this is the parabola -(0.2x - 1.5)^2 + 2.25
+		let step = (20 / totalLines) * num;
+		let prop_y = (-((0.15 * step - 1.5) * (0.15 * step - 1.5)) + 2.25) / 2.25;
+		return [`${step * 5}%`, `${prop_y * 100}%`];
+	}
+
+	function returnRotation(num: number) {
+		// this is the derivative of the aforementioned parabola turned into degrees of rotation
+		let step = (15 / totalLines) * num;
+		return Math.atan(-(2 * step - 15) / 25);
+	}
+
+	if (curTime > hackEndTime) {
+		return (
+			<div className={styles.countdownWrapper}>
+				<Image src={bg_map} alt="bg_map" />
```

Comment:
suggestion (repeating): change the alt to something like "background map for countdown"